science/hacktv/files/destdir-fix.diff

```diff
@@ -0,0 +1,27 @@
+--- Makefile.orig	2022-10-09 22:13:57.000000000 +0100
++++ Makefile	2022-10-09 22:41:52.000000000 +0100
+@@ -1,5 +1,5 @@
+ 
+-CC      := $(CROSS_HOST)gcc
++CC      := $(CROSS_HOST)$(MACPORTS_CC)
+ PKGCONF := $(CROSS_HOST)pkg-config
+ CFLAGS  := -g -Wall -pthread -O3 $(EXTRA_CFLAGS)
+ LDFLAGS := -g -lm -pthread $(EXTRA_LDFLAGS)
```

Comment:
By using `?=` here instead of `:=` you may enjoy MacPorts cflags which probably makes this port universal.

Reply:
Are you able to do this? I'm not "fsphil" (the author of that makefile) so I don't really want to make changes to his code that I don't fully understand. If you feel qualified to make those changes, I suppose you can.

What you say ("which probably makes this port universal") certainly sounds good!   It's just a bit beyond my personal competence level to judge ....

Reply:
OK - I've just looked up on google what the difference between := and ?= is.  Based on that I can see that what you say sounds plausible ... ish.

Reply:
changed